### ISSUE TITLE
Nov. Design Audit: Public facing enhancements

### ISSUE THREAD

**Issue by AvivaOskow** [18F/crt-portal#772]
## Landing

**DESKTOP**

#### DOJ seal and name
is wonky, seal looks too small in comparison to name, spacing between "U.S. Dept.." and larger "Civil Rights Division" is too large, and first line looks too big, Merriweather still looking extra bold/chunky
- [ ] Change "U.S. Department of Justice" to 12px
- [ ] reduce spacing between lines
- [ ] Make seal larger (60px works)
- [ ] Reduce spacing between seal and words to 8px

#### Currently:
![Screen Shot 2020-11-06 at 1.48.29 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/1c533436-2b57-470e-9e48-6d93e6f6924b)

#### Should be
![Screen Shot 2020-11-06 at 1.56.04 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/dcafe778-9f18-4bb4-bb86-770600f150ef)

**MOBILE**

- Same, looks wonky (need to see how this looks live because in my sketch file the font size is very small but when I change it in the inspector my pixels are too small, so let's start with this)
- [ ] Change "U.S. Department of Justice" to 12px
- [ ] Change "Civil Rights Division" to 18px
- [ ] Reduce spacing between lines 
- [ ] Make seal larger (45px should work)

#### Currently:
![Screen Shot 2020-11-06 at 2.13.02 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/cc30c964-33bc-4a9b-9a8d-31bd850333d3)

#### Should be
![Screen Shot 2020-11-06 at 2.13.39 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/a6c6259c-c454-4cb9-8eb2-c011df246e09)

#### Primary Concern pop-up
needs some design love:

- [ ] Blue tab is currently using # 005EA2, which is not one of our colors. Suggest changing to our blue-warm-80v # 162E51 or blue-50 # 2378C3
- Module box (same for other pages with it):
- [ ] Round the corners to match the cards in form (5px)
- [ ] Radio button style should match the form (using wrong color now)
- [ ] Button is using wrong blue as well, should match form

#### Currently (with wrong color blue and stylings):
![Screen Shot 2020-11-06 at 2.19.37 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/ff3cf3b2-d9a6-4b89-b417-ea7809bdbde8)
![Screen Shot 2020-11-06 at 2.19.20 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/a425e73b-cbe1-4ce1-aa89-d1b485cd738d)

#### Personal characteristics
still is missing space between card header and question
- [ ] Add spacing like on other cards (32px I believe)

#### Currently
![Screen Shot 2020-11-06 at 2.22.10 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/bcda0462-5bb2-40a3-a812-85bb5e3e85b2)

#### Should be
![Screen Shot 2020-11-06 at 2.22.46 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/3bd5452d-31b1-41e3-8645-42308535ec7a)

### Date 
- [ ] Can we add the USWDS date picker? We use it on the CRT side

#### Currently
![Screen Shot 2020-11-06 at 2.25.36 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/e03c8c0f-36c9-40a8-99e6-67ad2ef68528)

#### Could add (_used in table filters_)
![Screen Shot 2020-11-06 at 2.26.25 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/3ccf744a-1e41-4eac-b54b-85a782950c34) 

#### Personal description
- [ ] "happened" has a weird line break, I think because the “*required” makes it run on to 2 lines, but could just that one be on the second line?

#### Currently
![Screen Shot 2020-11-06 at 2.29.16 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/03c30147-d79b-40e0-9fd8-7eafe106b8eb)

#### Progress bar
- [ ] Too much space between progress bar and first white box (originally was supposed to say name of page)
**Solution:** Add the page name in so folks know what the page is - or - we make the space smaller

#### Currently 
![Screen Shot 2020-11-06 at 2.49.28 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/145cde46-5909-4a63-9e89-3a1a9790cb4d)

#### Could be
**With progress bar name added**
![Screen Shot 2020-11-06 at 2.51.28 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/7df45da6-4132-4b91-a1a9-8e44304af525)

**With less space**
![Screen Shot 2020-11-06 at 2.51.44 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/f3e1b56e-8665-4e97-8ba0-5be428d67edb)

### Review page
- [ ] Should we remove the “submit” button at top? We discussed this being a strange interaction because we are asking them to review but putting the button there which would encourage them to just click without looking potentially, whereas if it's at the bottom they at least have to scroll

#### Currently
![Screen Shot 2020-11-06 at 2.29.16 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/b18f1781-16cd-472f-81e7-fb97831b68f0)

#### Headers
- [ ] Can we add our little line dividers here like on the homepage? Between the large Merriweather headers ("What to expect, what you can do next") and the content right below?

#### Currently
![Screen Shot 2020-11-06 at 2.33.15 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/26ea105b-8456-4fe8-b9f5-c283edd7f476)

#### Should be
![Screen Shot 2020-11-06 at 2.36.56 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/d4993c13-cad9-4eca-be1b-5f51e7906e6b)

#### Examples from landing
![Screen Shot 2020-11-06 at 2.39.00 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/f9c3ce09-8b29-4ac0-98eb-c9944cac5685)

#### Print?

- [ ] Can we add a print button or an option to email themselves the confirmation page? I know this was out of scope before but wondering if it could be added soon

**Comment by cantsin** [comment]
Looked at `happened` quickly, I think this is a consequence of how the html breaks. If I resize the window, I can get the break so I don't think this one is fixable, unfortunately. (But do correct me if I'm not understanding this issue correctly)

**Comment by timoballard** [comment]
@AvivaOskow - clarifications on [DOJ seal and name section ](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/848):
- under mobile, you mentioned changing the text size to 12px and 14px for the two lines, but that appears to be what they're set to currently -- your "should be" image would suggest that you're looking for a bigger size difference between the two lines, perhaps 12px / 22px (pictured):

![image.png](https://images.zenhubusercontent.com/5f6b6d45a696d381a6f600f9/18ad96b7-a2ee-4d32-b866-51cd42e70734)

- the logo is currently 48px and doesn't change size when switching between desktop and mobile - it's simple enough to do what you described (45px for mobile, 60 for desktop), just want to make sure that's what you want. 

Thanks!

**Comment by AvivaOskow** [comment]
@timoballard Yeah the idea is that the first line is proportionally smaller than the second, but I'm wondering your thoughts on whether it would be better to just export PNGs of the logo + seal for desktop and mobile instead of trying to code the pieces individually? If we do go with individually though, yes the sizes I shared for the seal pixels should be correct (I'll have to check it after we make the change though to see if it worked out the way I meant it too). Thank you!

**Comment by timoballard** [comment]
Sure thing - it can be fiddly to code the text separately, but for accessibility and SEO reasons I think it's the way to go. I'll get these changes in and pushed up to a location that you can access asap!
